pxb-80/Dockerfile

```diff
@@ -1,32 +1,56 @@
-FROM centos:7
+FROM centos:8
 
+LABEL org.label-schema.schema-version="1.0"
+LABEL org.label-schema.name="Percona XtraBackup"
+LABEL org.label-schema.vendor="Percona"
+LABEL org.label-schema.description="Percona XtraBackup tools provide a method of performing a hot backup of your MySQL data while the system is running. \
+Percona XtraBackup is a free, online, open source, \
+complete database backups solution \
+for all versions of Percona Server for MySQL and MySQL®."
+LABEL org.label-schema.license="GPLv3"
+
+LABEL org.opencontainers.image.title="Percona XtraBackup"
+LABEL org.opencontainers.image.vendor="Percona"
+LABEL org.opencontainers.image.description="Percona XtraBackup tools provide a method of performing a hot backup of your MySQL data while the system is running. \
+Percona XtraBackup is a free, online, open source, \
+complete database backups solution \
+for all versions of Percona Server for MySQL and MySQL®."
+LABEL org.opencontainers.image.license="GPLv3"
```

Comment:
As I can see it should be v2

Name         : percona-xtrabackup-80
Version      : 8.0.13
Release      : 1.el8
Architecture : x86_64
Size         : 13 M
Source       : percona-xtrabackup-80-8.0.13-1.el8.src.rpm
Repository   : tools-release-x86_64
Summary      : XtraBackup online backup for MySQL / InnoDB
URL          : http://www.percona.com/software/percona-xtrabackup
License      : GPLv2
Description  : Percona XtraBackup is OpenSource online (non-blockable) backup solution for InnoDB and XtraDB engines